templates/shortcode-donor-wall.php

```diff
@@ -117,10 +119,16 @@
                             $excerpt = trim($excerpt, '.!,:;');
 
                             echo "<p class='give-donor-content__excerpt'>$excerpt &hellip;
-                                    <span> <a class='give-donor-content__read-more' style='color: {$primary_color}'> $read_more_text </a></span>
+                                    <span> <a class='give-donor-content__read-more' style='color: $primary_color'> $read_more_text </a></span>
                                    </p>";
 
                             echo "<p class='give-donor-content__comment'> $comment </p>";
+
+                        }
+                        else     {
+                            echo "<p class='give-donor-content__excerpt'>
```

Comment:
@JoshuaHungDinh for full comment, you can use `give-donor-content__comment` class.